sql/system_variables.go

```diff
@@ -66,7 +66,7 @@ type SystemVariable struct {
 
 // globalSystemVariables is the underlying type of SystemVariables.
 type globalSystemVariables struct {
-	mutex *sync.RWMutex
+	mutex      *sync.RWMutex
```

Comment:
Might want to merge / rebase master, these formatting changes are already in master